plasmapy/formulary/tests/test_parameters.py

```diff
@@ -365,6 +365,44 @@ def test_thermal_speed():
         thermal_speed(1 * u.MK, particle="p").si.value, 128486.56960876315
     )
 
+    # Explicitly check all three modes and dimensionalities
+    # ndim = 1
+    assert np.isclose(thermal_speed(T_e, method="most_probable", ndim=1).si.value, 0.0)
+
```

Comment:
```suggestion

    # Regression tests start here!
```
I'm bad at keeping track of which of our tests are known value comparisons (probably a minuscule amount) and which are just regression tests, so while I'm looking at this file, lemme add this note. This sort of thing hurt us a bit when astropy changed their current constant values. The implementation is fine, though.